src/CollapsibleCard/index.js

```diff
@@ -0,0 +1,157 @@
+          {title}
+        </h4>
+        <div
+          className={cc([
+            !isDisabled ? "subtitle" : "subtitle--disabled",
+            "padding--left--l",
+            "padding--bottom--l",
+            "margin--top--xxs",
+          ])}
+        >
+          {subtitle}
+        </div>
+      </div>
+      {helperText && (
+        <div style={{ display: "flex" }}>
+          <div
+            className="collapsible-card--helperText padding--right--l fontSize--s"
+            style={{ alignSelf: "center" }}
+          >
+            {helperText}
+          </div>
+        </div>
+      )}
+    </div>
+  );
+
+  return isOpen && !isDisabled ? (
+    <div
+      className={cc([
+        "collapsible-card--content-card",
+        {
+          "content-card--error": hasError,
+          "content-card--hover": !disableHover && hover,
+        },
+        "rounded--bottom",
+        "bgColor--white",
+      ])}
+    >
+      <div
+        className="collapsible-card--title-expanded"
+        role="button"
+        tabIndex={0}
+        onKeyUp={({ key }) => {
+          if (key === "Enter") onTitleContainerClick(false, "close");
+        }}
+        aria-expanded="true"
+        onClick={() => onTitleContainerClick(false, "close")}
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+      >
+        {titleContainerJSX}
+      </div>
+      <div className="padding--all--l">{children}</div>
+    </div>
+  ) : (
+    <div
+      className={cc([
+        "collapsible-card--content-card",
+        {
+          "content-card--error": hasError,
+          "content-card--disabled": isDisabled,
+          "content-card--hover": !disableHover && hover,
+        },
+        "content-card--closed",
+        "rounded--all",
+        "bgColor--white",
+      ])}
+      role="button"
+      tabIndex={0}
+      onKeyUp={({ key }) => {
+        if (key === "Enter") onTitleContainerClick(isDisabled, "open");
+      }}
+      aria-expanded="false"
+      aria-disabled={isDisabled ? "true" : "false"}
+      onClick={() => onTitleContainerClick(isDisabled, "open")}
+    >
+      {titleContainerJSX}
+    </div>
+  );
+};
+
+CollapsibleCard.propTypes = {
+  /** Accepts any content as children */
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node),
+  ]).isRequired,
+  /** Card title */
+  title: PropTypes.string,
+  /** Card subtitle */
+  subtitle: PropTypes.string,
+  /** Card helper text, placed on the right side of the title container */
+  helperText: PropTypes.node,
+  /** Controls whether card is opened */
+  isOpen: PropTypes.bool.isRequired,
+  /** Callback to handle user opening card */
+  onOpen: PropTypes.func,
+  /** Callback to handle user closing card */
+  onClose: PropTypes.func,
+  /** Disabled cards are greyed out and do not open */
+  isDisabled: PropTypes.bool,
+  /** Callback to handle user clickling on disabled card */
+  onDisabledClick: PropTypes.func,
```

Comment:
**Question**

Do we know of any use cases where we need this callback? We could remove this prop for now and put it back only if we need it. If we refactor `onOpen`/`onClose` into a single `onClick` callback, we could simply not call `onClick` when the card is disabled.

Reply:
Yes--currently, the way we're using it is to show a toast notification when you try to open a disabled card if this prop is passed. This is relevant for a page with multiple cards where one card is only enabled after a specific card is already completed (which is how we currently use this).